org.sixtyninestudios.kublit.yml

```diff
@@ -0,0 +1,38 @@
+app-id: org.sixtyninestudios.kublit
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: Kublit.sh
+finish-args: 
+  - --share=ipc
+  - --socket=x11 # <-- Godot needs X11
+  - --socket=pulseaudio
+  - --device=dri
+modules:
+  - name: Kublit
+    buildsystem: simple
+    build-commands:
+      - install -D Kublit.sh /app/bin/Kublit.sh
+      - install -D kubliticon.png /app/share/icons/kubliticon.png
+      - install -D org.sixtyninestudios.kublit.desktop /app/share/applications/org.sixtyninestudios.kublit.desktop
+      - install -D org.sixtyninestudios.kublit.metainfo.xml /app/share/metainfo/org.sixtyninestudios.kublit.metainfo.xml
+      - install -D KublitRelease.x86_64 /app/lib/Kublit/KublitRelease.x86_64
+      - tar -xf Getris-master.tar.xz --directory /app/lib/Kublit/
+    sources:
+      - type: file
+        url: https://archive.org/download/kublit-release.x-86-64_202303/KublitRelease.x86_64
+        sha256: c22ba93c23f48ecabbc788ffed42868a5ab7d66b57757c50967ec95e026a5d5d
+        
+      - type: file
+        path: Getris-master.tar.xz
```

Comment:
this file SHOULD NOT be in this repository.

Reply:
It does, it's nessacary for the game to function

Reply:
no it should not. It should download from wherever it comes from.

Reply:
The tar.xz file is there to extract the dictionary with all the contents necessary for the game to rin

Reply:
This is not part of what should be in this repository.

Reply:
why? The building process worked completely fine with it

Reply:
but the maintainers are not.